completions/xrandr

```diff
@@ -1,62 +1,165 @@
 # bash completion for xrandr                               -*- shell-script -*-
 
+_xrandr_outputs()
+{
+    "$1" 2>/dev/null | awk '/connected/ {print $1}'
+}
+
+_xrandr_monitors()
+{
+    "$1" --listmonitors 2>/dev/null | sed -ne 's/.* [0-9]*: [+\*]*\([^ ]*\).*/\1/p'
```

Comment:
`command sed` instead of plain `sed` everywhere, see examples in existing completions.

Reply:
Also, `test/runLint` might catch some non-portable sed etc uses, be sure to run it in case you didn't yet.

Reply:
> `command sed` instead of plain `sed` everywhere, see examples in existing completions.

Should `awk` be prefixed by `command` as well?

Reply:
No, we currently do it generally for grep, ls, and sed only. (Ref test/runLint)

Reply:
Done.